src/Table/__tests__/Table-test.js

```diff
@@ -44,17 +44,6 @@ describe('Table', function () {
       .toEqual(4);
   });
 
-  it('should render the proper number of rows', function () {
```

Comment:
We don't have `getRows` anymore, that was only used for when we differentiated between scrollbar/no scrollbar. We always use `VirtualList` now